irr/include/WeightBuffer.h

```diff
@@ -0,0 +1,62 @@
+// Luanti
+// SPDX-License-Identifier: LGPL-2.1-or-later
```

Comment:
Would it be OK for you to use the Irrlicht license here to stay consistent with the other header file that makes use of this one?